src/stringparse.rs

```diff
@@ -73,3 +73,9 @@ pub fn handle_unicode_codepoint(
     let offset: usize = codepoint_to_utf8(code_point, dst_ptr);
     Ok((offset, src_offset))
 }
+
+// Holds backslashes and quotes locations.
+pub struct ParseStringHelper {
```

Comment:
I think this is arm specific isn't it? If so it should live in the arm sub directory.

Reply:
@Licenser one thing that's neat about the `find_quote_mask_and_bits()` function is that it encapsulates platform-specific logic in the main deserializer function, maybe we should consider adopting the pattern on the x86 side? (Which will slim down code...)

Reply:
@Licenser the code between the architectures is converging -- I have a nifty refactoring that I'll send as another PR.